### PR TITLE
oded/act 44

### PR DESCRIPTION
**Pull request by odeds**

### CODE REVIEW

File: src/components/analytics/services/znkAnalyticsSrv.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+(function (angular) {
+    'use strict';
+
+    var _eventsConst = {
```
Comment (assafshp) [inline src/components/analytics/services/znkAnalyticsSrv.js]: good !

File: src/components/popUp/popUpSrv.js [hunk 2]
```diff
@@ -45,10 +45,8 @@
                                 '<div ng-repeat="button in ::d.buttons" class="button-wrapper">' +
                                     '<div class="btn" ' +
                                              'ng-click="d.btnClick(button)" ' +
-                                             'ng-class="button.type" ' +
-                                             'analytics-on="click" ' +
```
Comment (assafshp) [inline src/components/popUp/popUpSrv.js]: did you make sure we no longer have analytics-on in our code ?

File: src/components/analytics/services/znkAnalyticsUtilSrv.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                    previousValue.wrong++;
+                }
+            } else {
+                previousValue.skip++;
+            }
+            return previousValue;
+        },{ correct: 0, wrong: 0, skip: 0 });
+    }
+
+    angular.module('znk.infra.analytics').service('znkAnalyticsUtilSrv', ['$log', function ($log) {
```
Comment (assafshp) [inline src/components/analytics/services/znkAnalyticsUtilSrv.js]: why not in a diffrent file?

Comment (odeds) [inline src/components/analytics/services/znkAnalyticsUtilSrv.js]: the znkAnalyticsUtilSrv is in a difrrent file from znkAnalyticsSrv, but this helper function are part of the znkAnalyticsUtilSrv

File: src/components/popUp/popUpSrv.js (hunk 2 above)
Comment (odeds) [inline src/components/popUp/popUpSrv.js]: find some in the mobile template for znk-exercise, removed.

**Pull request merged**